.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+--------
+Pull Requeset Description
+--------
+##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
+
+
+##### What are the significant changes in functionality due to this change request?
+
+
+----------------
+For Change Control Board: Change Request Review
+----------------
+The following review must be completed by an authorized member of the Change Control Board.
+- [ ] 1. Review all computer code.
+- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
+- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://hpcgitlab.inl.gov/idaholab/raven/wikis/Developer_Information#python-code-standard) for details.
```

Comment:
Ha, yes, this is true.  Do you have a preference?  I like the wiki myself.  I can do that if you want.